datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/saved_views.py

```diff
@@ -53,13 +54,15 @@ def saved_views(check):
         saved_views, _ = get_assets_from_manifest(integration, 'saved_views')
 
         for saved_view in saved_views:
-
+            display_queue = []
             # all saved views must be json
             try:
                 view = load_saved_views(saved_view)
             except json.JSONDecodeError as e:
                 errors = True
-                echo_failure(f"{integration} saved view is not valid json: {e}")
+                message = f"{integration} saved view is not valid json: {e}"
+                echo_failure(message)
```

Comment:
Should this also be in a `display_queue`?

Reply:
No, after this error, the validation continues to run through other integrations before display queue renders the messages.